lighthouse-core/gather/gather-runner.js

```diff
@@ -104,7 +107,8 @@ class GatherRunner {
       .then(_ => driver.registerPerformanceObserver())
       .then(_ => driver.dismissJavaScriptDialogs())
       .then(_ => resetStorage && driver.clearDataForOrigin(options.url))
-      .then(_ => gathererResults.UserAgent = [driver.getUserAgent()]);
+      .then(_ => gathererResults.UserAgent = [driver.getUserAgent()])
```

Comment:
heh, want to move this line above begin emulation while you're in here updating the docs :)